# SteeringSpeeds.msg - 转向速度反馈
# 轮速信息
float64 wheel_01        # 1号轮速度 (单位: m/s)
float64 wheel_02        # 2号轮速度 (单位: m/s)
float64 wheel_03        # 3号轮速度 (单位: m/s)
float64 wheel_04        # 4号轮速度 (单位: m/s)